WcaOnRails/.rubocop.yml

```diff
@@ -5,9 +5,7 @@ AllCops:
   DisplayCopNames: true
   Exclude:
     - 'node_modules/**/*'
-    - 'bin/webpack-dev-server'
-    - 'bin/webpack'
-    - 'bin/yarn'
+    - 'bin/**/*'
```

Comment:
Rubocop complained about top-level `include` calls in some `bin/` scripts. There's no `class`/`module` in those files, so it would be hard to make Rubocop happy. More importantly, files in the `bin/` directory are from libraries, so I don't think we need them to comply with our code guidelines.